charts/kong/templates/deployment.yaml

```diff
@@ -36,6 +36,9 @@ spec:
       labels:
         {{- include "kong.metaLabels" . | nindent 8 }}
         app.kubernetes.io/component: app
+        {{- if .Values.podLabels }}
+        {{ toYaml .Values.podLabels | nindent 8 }}
+        {{- end }}
```

Comment:
Please update an existing test file to include a test case.

Reply:
This should be all set.  The following snippet from the test showing the app and environment labels added.

```bash
Node:         chart-testing-control-plane/172.17.0.2
Start Time:   Fri, 24 Apr 2020 03:22:06 +0000
Labels:       app=kong
              app.kubernetes.io/component=app
              app.kubernetes.io/instance=kong-5n1v8lzb7n
              app.kubernetes.io/managed-by=Tiller
              app.kubernetes.io/name=kong
              app.kubernetes.io/version=2
              environment=test
```